modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -1240,13 +1002,14 @@ static void run_absdiffc(Buffer &dst, const View &src, const float scalar[])
 
     int width = dst.length();
     int chan = dst.meta().chan;
+    const int length = width * chan;
```

Comment:
just for interest: why all types is signed? is it by historical reasons?

Reply:
Yes, it is for history :-)